### ISSUE TITLE
build error mumble-client/index.js (webrtc branch)

### ISSUE THREAD

**Issue by streaps** [Johni0702/mumble-web#42]
this happens if you do a `git checkout webrtc` right after `git clone` and try to build it.

```
# npm run build

> mumble-web@0.5.1 build /root/repos/mumble-web
> webpack && [ -f dist/config.local.js ] || cp app/config.local.js dist/

Hash: 11529420d67c1be39a0a
Version: webpack 1.15.0
Time: 7243ms
                                Asset       Size  Chunks             Chunk Names
 296b612d2847da52749549f73d0c3664.svg    1.29 kB          [emitted]  
 50dddae19e7bf601b168f46a1303674b.svg     2.4 kB          [emitted]  
[...]
                             theme.js    78.7 kB       3  [emitted]  theme
   [0] multi index 40 bytes {1} [built]
    + 234 hidden modules

ERROR in ./~/mumble-client/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./lib/client.js in /root/repos/mumble-web/node_modules/mumble-client
 @ ./~/mumble-client/index.js 1:17-43

```

**Comment by Johni0702** [comment]
Did you run `npm install` between checkout and build?

**Comment by streaps** [comment]
> Did you run `npm install` between checkout and build?

Yes, I did:

git clone https://github.com/johni0702/mumble-web
cd mumble-web
git checkout webrtc
npm install
npm run build
...

**Comment by Johni0702** [comment]
Node 0.8 and 0.10 are really, really old. Any Node version below 6.x is no longer maintained.
I'm currently using 10.x for development and I think at one point it was working on 8.x as well in case you can't use 10.x.
I've never tried 6.x or anything more recent than 10.x.

**Comment by streaps** [comment]
Sorry, I meant 8.x and 10.x

**Comment by Johni0702** [comment]
Indeed, looks like the `mumble-client` dependency wasn't build correctly when installed from git (as it has to be for the webrtc branch).

Above commit updates `mumble-client` to a commit which builds properly and therefore should fix the issue.
Thanks for your reports.

**Comment by streaps** [comment]
I can confirm that it did fix the issue :)

**Comment by streaps** [comment]
I'm getting this error again with the webrtc branch.

**Comment by Johni0702** [comment]
No issues for me: https://gist.github.com/Johni0702/8c1043f2a99f02a48ff9efed8abd5a70

**Comment by streaps** [comment]
running `npm install` solved the issue too (see #92)
